recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,278 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+{% set build_num = 0 %}
+{% set name_ = name|lower %}
```

Comment:
This is an unnecessary duplicate. I would use `name|lower` where required.

Reply:
I am actually using that variable in multiple places. So, decided to use a short hand (at least for the time being). I will use a multi-cursor later on to replace all `name_` occurrences with the more verbose `name|lower`.